app/src/main/java/de/xikolo/App.kt

```diff
@@ -102,4 +117,29 @@ class App : Application() {
         CookieManager.getInstance().flush()
     }
 
+    private fun updateShortcuts() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
```

Comment:
Should e a feature config. Also, the observer code should only be called is this condition is fulfilled.

Reply:
Can you specify feature config? Google was not very helpfull.
Is it something like @RequiresApi?